src/flatland/ordered/map.clj

```diff
@@ -86,7 +86,8 @@
     (.equiv this other))
   (hashCode [this]
     (reduce (fn [acc ^MapEntry e]
-              (let [k (.key e), v (.val e)]
+              (let [k (if-some [k (.key e)] k 0)
```

Comment:
OK, thanks; I'll add a unit test for that; I didn't realize IEquiv used hashCode as a heuristic. Or does that only matter when placing the maps in (associative) collections?